pixi.toml

```diff
@@ -25,7 +25,7 @@ deploy-dev = "mike deploy --push dev devel"
 
 [dependencies]
 openssl = "3.*"
-rust = "~=1.81.0"
+rust = ">=1.83.0,<1.84"
```

Comment:
That doesn't seem to match the rest of this PR, which is on 1.84 already (but I guess 1.84 isn't packaged yet...)

Reply:
made them all 1.83